packages/Rakefile

```diff
@@ -163,6 +163,26 @@ class GroongaPackageTask < PackagesGroongaOrgPackageTask
   def built_package_n_split_components
     3
   end
+
+  def download_packages(target_namespace)
+    return if target_namespace == :windows
+    super
+  end
+
+  def prepare(target_namespace)
+    return unless target_namespace == :windows
+    
+    repositories_dir = download_repositories_dir(target_namespace)
+    mkdir_p(repositories_dir)
+
+    File.open("#{repositories_dir}/.htaccess", "w") do |htaccess|
+      __send__("#{target_namespace}_targets").each do |target|
+        redirect_url = built_package_url(target_namespace, target)
+        redirect_target = target.gsub(@version, "latest")
+        htaccess.puts("RewriteRule #{redirect_target} #{redirect_url}")  
```

Comment:
I have changed to use `RedirectMatch` instead of `Redirect` in order to simplify a path specification in source code.

The new `.htaccess` is as below.

```
RedirectMatch ^(.+)/groonga\-latest\-x64\-vs2022\.zip$ https://github.com/groonga/groonga/releases/download/v12.0.9/groonga-12.0.9-x64-vs2022.zip
RedirectMatch ^(.+)/groonga\-latest\-x64\-vs2022\-with\-vcruntime\.zip$ https://github.com/groonga/groonga/releases/download/v12.0.9/groonga-12.0.9-x64-vs2022-with-vcruntime.zip
RedirectMatch ^(.+)/groonga\-latest\-x64\-vs2019\.zip$ https://github.com/groonga/groonga/releases/download/v12.0.9/groonga-12.0.9-x64-vs2019.zip
RedirectMatch ^(.+)/groonga\-latest\-x64\-vs2019\-with\-vcruntime\.zip$ https://github.com/groonga/groonga/releases/download/v12.0.9/groonga-12.0.9-x64-vs2019-with-vcruntime.zip
RedirectMatch ^(.+)/groonga\-latest\-x86\-vs2019\.zip$ https://github.com/groonga/groonga/releases/download/v12.0.9/groonga-12.0.9-x86-vs2019.zip
RedirectMatch ^(.+)/groonga\-latest\-x86\-vs2019\-with\-vcruntime\.zip$ https://github.com/groonga/groonga/releases/download/v12.0.9/groonga-12.0.9-x86-vs2019-with-vcruntime.zip
```

Reply:
As a first, the rule below was incorrect because it redirects `http://packages.groonga.org/windows/packages/groonga-latest-x64-vs2022.zip` to `http://packages.groonga.org/windows/packages/https://github.com/groonga/groonga/releases/download/v12.0.8/groonga-12.0.8-x64-vs2022.zip`. We need specify absolute path for old URL.

`RewriteRule groonga-latest-x64-vs2022.zip https://github.com/groonga/groonga/releases/download/v12.0.8/groonga-12.0.8-x64-vs2022.zip`

If we use `Redirect`, the rule will be as below.

```
Redirect /windows/groonga/groonga-latest-x86-vs2019.zip https://github.com/groonga/groonga/releases/download/v12.0.9/groonga-12.0.9-x86-vs2019.zip
```

The reason why I used `RedirectMatch` was just because to avoid creating `/windows/groonga` in source code.

Reply:
How about just prepending `/windows/groonga/` instead of using regular expression?
`/windows/groonga/` will not be changed.

Reply:
Or we can use `/#{target_namespace}/#{@package}/`.

Reply:
I see. Thanks.

> Or we can use `/#{target_namespace}/#{@package}/`.

I have changed to use it.